scalr/data_source_workspace_ids_test.go

```diff
@@ -72,48 +71,58 @@ func TestAccScalrWorkspaceIDsDataSource_wildcard(t *testing.T) {
 	})
 }
 
-func testAccScalrWorkspaceIDsDataSourceConfig_basic(rInt int) string {
+func testAccScalrWorkspaceIDsDataSourceConfigBasic(rInt int) string {
 	return fmt.Sprintf(`
-resource "scalr_workspace" "foo" {
+resource scalr_environment test {
+  name       = "test-env-%d"
+  account_id = "acc-svrcncgh453bi8g"
+}
+
+resource scalr_workspace foo {
   name           = "workspace-foo-%d"
-  environment_id = "existing-env"
+  environment_id = scalr_environment.test.id
 }
 
-resource "scalr_workspace" "bar" {
+resource scalr_workspace bar {
   name           = "workspace-bar-%d"
-  environment_id = "existing-env"
+  environment_id = scalr_environment.test.id
 }
 
-resource "scalr_workspace" "dummy" {
+resource scalr_workspace dummy {
   name           = "workspace-dummy-%d"
-  environment_id = "existing-env"
+  environment_id = scalr_environment.test.id
 }
 
-data "scalr_workspace_ids" "foobar" {
-  names          = ["${scalr_workspace.foo.name}", "${scalr_workspace.bar.name}"]
-  environment_id = "existing-env"
-}`, rInt, rInt, rInt)
+data scalr_workspace_ids foobar {
+  names          = [scalr_workspace.foo.name, scalr_workspace.bar.name]
+  environment_id = scalr_environment.test.id
+}`, rInt, rInt, rInt, rInt)
 }
 
-func testAccScalrWorkspaceIDsDataSourceConfig_wildcard(rInt int) string {
+func testAccScalrWorkspaceIDsDataSourceConfigWildcard(rInt int) string {
```

Comment:
I changed all helper functions to camelCase, according to naming conventions and linters.